src/main/java/uk/gov/hmcts/dm/config/WebConfig.java

```diff
@@ -1,13 +1,26 @@
 package uk.gov.hmcts.dm.config;
 
+import org.springframework.beans.propertyeditors.CustomDateEditor;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.hateoas.config.EnableHypermediaSupport;
 
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+
 /**
  * Created by pawel on 22/08/2017.
  */
 @Configuration
 @EnableHypermediaSupport(type = EnableHypermediaSupport.HypermediaType.HAL)
 public class WebConfig {
 
+    private final DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssX", Locale.UK);
```

Comment:
`SimpleDateFormat` is not thread safe - see https://stackoverflow.com/questions/6840803/why-is-javas-simpledateformat-not-thread-safe